docs/usage/applications/access_pod_from_local/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title:  Access a port of a pod locally
```

Comment:
Why do you think this guides needs to be in g/g?

Reply:
Same as above.

Reply:
This is something documented in the k8s website since long time, see https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/, why we should maintain our own version such document ...